pkg/broker/broker.go

```diff
@@ -254,16 +267,18 @@ func encodePlan(v dynamicplans.Plan) (string, error) {
 	b64 := base64.NewEncoder(base64.StdEncoding, b)
 	err := json.NewEncoder(b64).Encode(v)
 	if err != nil {
-		return "", err
+		return "", errors.Wrap(err, "cannot marshal plan")
```

Comment:
interesting, wasn't aware of changes in`errors` regarding the wrapping.
Am I right that `errors.Wrap` will override the string message so the initial message will be lost (though the causing error will be saved internally). So loggers may miss that?
Maybe `errors.Errorf("..%w")` would be a better way?

Reply:
This `Wrap()` will actually append the inner message when `Error()` is called: https://godoc.org/github.com/pkg/errors#Wrap

Reply:
A real-life example of this would be the following error message:
`Service broker error: cannot find plan for instance "c20bdcea-6bf1-4804-ab06-bed618054e7d": cannot fetch instance: cannot find instance in maintenance DB(s): no instances found`

Here every `:` is basically a `Wrap()` call, going all the way to the top.